caffe2/core/context_gpu.h

```diff
@@ -357,14 +357,13 @@ struct CAFFE2_CUDA_API PinnedCPUAllocator final : public at::Allocator {
     at::DataPtr data_ptr;
     std::lock_guard<std::mutex> lock(CUDAContext::mutex());
     if (IsNUMAEnabled()) {
-      data_ptr = baseAllocator_.allocate(nbytes);
-      data = data_ptr.get();
+      data = baseAllocator_.naked_allocate(nbytes);
```

Comment:
Actually, we are in the process of merge pytorch and caffe2 allocator and we'll be using `Allocator*` for `baseAllocator_`, see: https://github.com/pytorch/pytorch/pull/14517/files#diff-6286b32ea83ee15c66db129928f27c42R343